src/containers/SearchBar.js

```diff
@@ -38,14 +38,15 @@ const SearchInput = styled.input`
   }
 `;
 
+//@todo move this to containers
```

Comment:
Can you kill this TODO now that the file is within containers? Or would you rather this specific function was in containers as its own file?